modules/market/src/main/java/com/opengamma/strata/market/curve/node/FixedIborSwapCurveNode.java

```diff
@@ -80,7 +87,26 @@ public static FixedIborSwapCurveNode of(FixedIborSwapTemplate template, Observab
    * @return a node whose instrument is built from the template using a market rate
    */
   public static FixedIborSwapCurveNode of(FixedIborSwapTemplate template, ObservableKey rateKey, double additionalSpread) {
-    return new FixedIborSwapCurveNode(template, rateKey, additionalSpread);
+    return of(template, rateKey, additionalSpread, null);
+  }
+
+  /**
+   * Returns a curve node for a Fixed-Ibor interest rate swap using the
+   * specified instrument template, rate key, spread and label.
+   *
+   * @param template  the template defining the node instrument
+   * @param rateKey  the key identifying the market data providing the rate for the node instrument
+   * @param additionalSpread  the additional spread amount added to the rate
+   * @param label  the label to use for the node, if null or empty an appropriate default label will be used
```

Comment:
Allowing a null value for the label feels a bit un-Strata. Particularly when there's an overload of the method that doesn't include the label. If you don't want to specify the label you can use the version with no label parameter, if you use the version with the label you should have to specify a valid label.